casepropods/family_connect_registration/tests.py

```diff
@@ -0,0 +1,136 @@
+import json
+import responses
+from django.apps import apps
+from casepro.cases.models import Case
+from casepro.test import BaseCasesTest
+from .plugin import RegistrationPodConfig, RegistrationPod
+
+
+class RegistrationPodTest(BaseCasesTest):
+    def setUp(self):
+        super(RegistrationPodTest, self).setUp()
+        self.contact = self.create_contact(self.unicef, 'test_id', "Mother")
+        self.msg1 = self.create_message(
+            self.unicef, 123, self.contact, "Hello")
+        self.case = Case.get_or_open(
+            self.unicef, self.user1, self.msg1, "Summary", self.moh)
+        self.url = 'http://example.com/registration/?mother_id=' + \
+            self.contact.uuid
+
+        self.pod = RegistrationPod(
+                apps.get_app_config('family_connect_registration_pod'),
+                RegistrationPodConfig({
+                    'index': 23,
+                    'title': "My registration Pod",
+                    'url': "http://example.com/registration/",
+                    'token': "test_token",
+                    'field_mapping': [
+                        {"field": "mama_name", "field_name": "Mother Name"},
+                        {"field": "mama_surname",
+                            "field_name": "Mother Surname"},
+                        {"field": "last_period_date",
+                            "field_name": "Date of last period"},
+                        {"field": "language",
+                            "field_name": "Language Preference"},
+                        {"field": "mama_id_type", "field_name": "ID Type"},
+                        {"field": "mama_id_no", "field_name": "ID Number"},
+                        {"field": "msg_receiver",
+                            "field_name": "Message Receiver"},
+                        {"field": "receiver_id", "field_name": "Receiver ID"},
+                        {"field": "hoh_name",
+                            "field_name": "Head of Household Name"},
+                        {"field": "hoh_surname",
+                            "field_name": "Head of Household Surname"},
+                        {"field": "hoh_id",
+                            "field_name": "Head of Household ID"},
+                        {"field": "operator_id", "field_name": "Operator ID"},
+                        {"field": "msg_type",
+                            "field_name": "Receives Messages As"},
+                    ]
+                }))
+
+    def registration_callback_no_matches(self, request):
+        headers = {'Content-Type': "application/json"}
+        headers = {'Authorization': "Token test_token"}
```

Comment:
Line 54 seems to be overwriting the assignment in line 53. The response probably doesn't need to have the token, just the `Content-Type`.